### ISSUE TITLE
Trying a fresh install, results in errors

### ISSUE THREAD

**Issue by samip5** [authelia/chartrepo#120]
I tried to install this chart, but it failed with the following error:
"spec.updateStrategy.rollingUpdate.maxSurge: Invalid value: intstr.IntOrString{Type:1, IntVal:0, StrVal:"25%"}: may not be set when maxUnavailable is non-zero"

What is the issue?

I have not set that, so my values are as follows:
```yaml
spec:
  interval: 5m
  chart:
    spec:
      # renovate: registryUrl=https://charts.authelia.com
      chart: authelia
      version: 0.6.3
      sourceRef:
        kind: HelmRepository
        name: authelia-charts
        namespace: flux-system
      interval: 5m
  values:
    fullnameOverride: authelia-dev-domain
    rbac:
      enabled: true
      serviceAccountName: authelia-dev
    domain: ${DEV_DOMAIN}
    ingress:
      subdomain: sso
      tls:
        enabled: true
        secret: networking/${DEV_DOMAIN}-tls
      traefikCRD:
        enabled: false

    configMap:
      enabled: true
      server:
        port: 9091
        read_buffer_size: 4096
        write_buffer_size: 4096

      log:
        level: info
        format: json

      theme: dark
      totp:
        issuer: ${DEV_DOMAIN}
        period: 30
        skew: 1

      authenication_backend:
        disable_reset_password: false

        ldap:
          enabled: true
          implementation: custom

          url: ldap://ak-outpost-ldap.security.svc.cluster.local
          start_tls: false
          base_dn: DC=ldap,DC=skylab,DC=fi
          username_attribute: "cn"
          additional_users_dn: "ou=users"
          user: cn=akadmin,dc=ldap,DC=skylab,dc=fi
```

**Comment by james-d-elliott** [comment]
This was fixed in 0.7.0 (0.7.7 is the latest), it can relatively easily be backported if you have a reason to use 0.6.x

**Comment by james-d-elliott** [comment]
Feel free to reopen if you're still having issues.
